Singer.API/Models/SearchResults.cs

```diff
@@ -0,0 +1,25 @@
+using Singer.DTOs;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Singer.Models
+{
+   public class SearchResults<T>
+   {
+
+      public SearchResults(IReadOnlyList<T> items, int totalItemsCount, int pageIndex)
+      {
+         Items = items;
+         TotalCount = totalItemsCount;
+         Start = pageIndex;
+         Size = items.Count;
+      }
+
+      public int TotalCount { get;  }
+      public int Start { get;  }
+      public int Size { get; }
+      public IReadOnlyList<T> Items { get; }
```

Comment:
Why is it an IReadOnlyList here while in the pagination model the type has changed to List?

Reply:
The `PaginationDTO` contains an IReadonlyList too, doesn't it?

Reply:
Looked wrong, my bad